src/main/java/edu/harvard/iq/dataverse/Dataverse.java

```diff
@@ -111,23 +111,23 @@ public void setDataverseType(DataverseType dataverseType) {
 
         public String getFriendlyCategoryName(){
        switch (this.dataverseType) {
-            case RESEARCHERS:
```

Comment:
We need to be careful that this (and other changes in this PR) are used by places in the app that do want the localized version. Maybe we should change the getIndexableCategoryName instead? (ideally, both it and this would call one method, and pass parameters to either get localized or default)

Reply:
@scolapasta  - refactored mehtods in Dataverse.java and FileUtil.java.  Please add your comments